.github/workflows/main.yaml

```diff
@@ -39,6 +50,7 @@ jobs:
         run: |
           git config user.name 'William Hernandez'
           git config user.email 'williamhe@microsoft.com'
+          git checkout doc/supportedPlatformVersions.md
```

Comment:
nit: what is this command doing? I typically associate `git checkout` with switching branches, but this command references a file, so just want to ensure I understand what's happening here

Reply:
@william-msft do you mind sharing an example of that bug? In theory, I feel like we should be updating that file as we release new versions which the code will support

Reply:
Agree with @pauld-msft -- we should quickly check out the bug in the generator to see why it's producing more supported versions than expected so that this cron job will help to accurately reflect the platform versions we support.

Reply:
Surprisingly I'm not able to reproduce locally. Maybe the github workflow triggers had an older version of the .`md` or something. I created a story just in case: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1664336

I've removed the `git checkout` now.